app/assets/stylesheets/provider/admin/_dashboard.scss

```diff
@@ -29,12 +29,9 @@
     font-weight: $font-weight-bold;
     color: $font-color;
     float: left;
-    max-width: line-height-times(10);
     margin: 0 0 0 0;
```

Comment:
Don't you want to add some right margin to make sure Overview doesn't get too close to the service name?

Reply:
Agree, if no right-margin this happens:
![no-margin](https://user-images.githubusercontent.com/13486237/48545968-67a80280-e8c7-11e8-9269-61f9fc025f8e.png)
We can do `margin: 0 2rem 0 0`

Reply:
it can't, it would wrap to next line.

Reply:
It depends on the size of the screen, If you resize at some point it could happen

Reply:
It depends on the size of the screen, if you resize at some point it could happen

Reply:
Thanks, you are both right. I need coffee…